app/src/main/java/com/example/keybindhelper/RecyclerViewAdapters/KeybindAdapter.java

```diff
@@ -215,17 +215,17 @@ private void showContextMenu(Keybind k, View view) {
             d.cancel();
             ArrowProvider ap=new ArrowProvider(context);
             ap.directionClicked=isUp -> {
-                int indx=k.group.keybinds.indexOf(k);
+                int index=k.group.keybinds.indexOf(k);
```

Comment:
Changed indx to index since it is not great practice to abbreviate. Source: https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/general-naming-conventions

"DO NOT use abbreviations or contractions as part of identifier names."